config/locales/en/placements/support/schools/mentors.yml

```diff
@@ -0,0 +1,12 @@
+en:
+  placements:
+    support:
+      schools:
+        mentors:
+          show:
+            caption: "Mentors - %{school_name}"
+            attributes:
+              mentors:
+                first_name: First name
+                last_name: Last name
```

Comment:
And these two

Reply:
resolved in [3852d92](https://github.com/DFE-Digital/itt-mentor-services/pull/274/commits/3852d927370d77ecb3dd252d00e4ec3e140b9852)